cosmic_ray/mutating.py

```diff
@@ -49,7 +47,8 @@ def activation_record(self):
         """
         return self._activation_record
 
-    def visit_mutation_site(self, node, op, num_mutations):
+    def visit_mutation_site(self, node, op,  # pylint: disable=invalid-name
```

Comment:
Was pylink marking `visit_mutation_site` as an invalid name?

Reply:
it's the 'op' :-) too short for argument name according to pylint